src/main/java/com/mineboundteam/minebound/block/BoomButton.java

```diff
@@ -0,0 +1,59 @@
+package com.mineboundteam.minebound.block;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Explosion;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+
+import java.util.Random;
+import java.util.UUID;
+
+public class BoomButton extends Block {
+
+    public BoomButton(Properties properties) {
+        super(properties);
+    }
+
+    @Override
+    public void onPlace(BlockState blockState, Level level, BlockPos blockPos, BlockState p_60569_, boolean p_60570_) {
+        // Schedule a tick for 60 ticks in the future (3 seconds, 20 ticks per second)
+        level.scheduleTick(blockPos, blockState.getBlock(), 60);
+    }
+
+    @Override
+    public void tick(BlockState p_60462_, ServerLevel level, BlockPos blockPos, Random p_60465_) {
+        // Runs when scheduled in onPlace and destroys the block
+        level.destroyBlock(blockPos, false);
+    }
```

Comment:
Took forever to figure out, but it's been changed to schedule a tick for destroying the block. Here is the new code.